### PR TITLE
Exposes API to infer release of people

### PR DESCRIPTION
**Pull request by terinpw**
- Created infer_release.py which contains an API to infer releases of people in the supplied region
- Moved helper methods from scraper_control to ingest_utils that are needed by infer_release
- Moved helper methods from scraper_utils into common_utils to prevent circular dependency in persistence (was throwing errors with my new test).

### CODE REVIEW

File: recidiviz/ingest/infer_release.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+from recidiviz.utils.params import get_values
+from recidiviz.persistence import persistence
+
+infer_release_blueprint = Blueprint('infer_release', __name__)
+
+
+@infer_release_blueprint.route('/release')
+@authenticate_request
+def infer_release():
+    log = logging.getLogger()
```
Comment (arian487) [inline recidiviz/ingest/infer_release.py]: This is unnecessary, you can just do logging.error which will use the default logger like you're getting here.

File: recidiviz/ingest/ingest_utils.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
+# Recidiviz - a platform for tracking granular recidivism metrics in real time
```
Comment (arian487) [inline recidiviz/ingest/ingest_utils.py]: Can you merge the ingest_info_utils with this.  Doesn't make sense having an ingest_utils and an ingest_info_utils.  I think you can take all of the code in ingest_info_utils and put it in here (the name makes more sense).

File: recidiviz/ingest/infer_release.py (hunk 1 above)
Comment (terinpw) [reply recidiviz/ingest/infer_release.py]: Cool done.

File: recidiviz/ingest/ingest_utils.py (hunk 2 above)
Comment (terinpw) [reply recidiviz/ingest/ingest_utils.py]: Ah didn't realize that file existed. Moved ingestion_info_utils code into ingest_utils (and same with tests).

**Pull request merged**